libafl/src/events/llmp.rs

```diff
@@ -1216,6 +1223,37 @@ where
         .launch()
 }
 
+/// Calling this function will tell the llmp broker that this client is exiting
+/// This should be called from the restarter not from the actual fuzzer client
+/// This function serves the same roll as the `LlmpClient.send_exiting()`
+/// However, from the the event restarter process it is forbidden to call `send_exiting()`
+/// (You can call it and it compiles but you should never do so)
+/// `send_exiting()` is exclusive to the fuzzer client.
+#[cfg(feature = "std")]
+pub fn notify_death(client_id: u32, broker_port: u16) -> Result<(), Error> {
```

Comment:
As discussed in the other guy, can we move this function into the event manager and remove both parameters?

Reply:
only client_id.
i cannot move broker_port because broker_port is not a member of LlmpEventManager

Reply:
Let's just make it a member?

Reply:
If we ever have another handshake mechanism than TCP we can still add special casing, but for now all LlmpEventManager de-facto have a port

Reply:
I mean, we could also consider either adding the port to the description, or add the port to the`existing_client_from_description` method. But maybe not urgent, then